src/main/java/com/faforever/client/rankedmatch/SearchLadderClientMessage.java

```diff
@@ -1,16 +1,17 @@
 package com.faforever.client.rankedmatch;
 
+import com.faforever.client.fa.RatingMode;
 import com.faforever.client.game.Faction;
 import com.faforever.client.game.KnownFeaturedMod;
 import com.faforever.client.remote.domain.ClientMessageType;
 
-public class SearchLadder1v1ClientMessage extends MatchMakerClientMessage {
+public class SearchLadderClientMessage extends MatchMakerClientMessage {
 
   private Faction faction;
 
-  public SearchLadder1v1ClientMessage(Faction faction) {
+  public SearchLadderClientMessage(RatingMode ratingMode, Faction faction) {
```

Comment:
It's called `mod` but it's really a matchmaker queue technical name.

Reply:
Yes. If you implement QueueType endpoint in rest api I will change it to some QueueType object.
https://github.com/FAForever/downlords-faf-client/pull/1786#discussion_r443147988